views.js

```diff
@@ -35,7 +35,7 @@ function big (state) {
   blit(screen, renderMessages(state, process.stdout.columns - 17 - 17, process.stdout.rows - HEADER_ROWS), 18, 3)
 
   // nicks pane
-  blit(screen, renderNicks(state, 16, process.stdout.rows - HEADER_ROWS), process.stdout.columns - 15, 3)
+  blit(screen, renderNicks(state, 15, process.stdout.rows - HEADER_ROWS), process.stdout.columns - 15, 3)
```

Comment:
We're saying that x coordinate should be `process.stdout.columns - 15` so should also slice using `15`.